packages/peregrine/lib/talons/MiniCart/__tests__/useItem.spec.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Consider a test case for the unmount cleanup effect (useItem:L14)

Reply:
I am still pondering if we even need that unmount logic. If the component is unmounting anyway, what's the point of setting the state? I must have forgotten to remove that piece of code

Reply:
I agree with removal; local state doesn't matter if you're unmounting anyway.

Reply:
Done removed.